.github/workflows/test.yaml

```diff
@@ -0,0 +1,38 @@
+name: test
```

Comment:
The e2e action also does a `make test`. Do we need a separate action to do it again?

Reply:
yes. It's intended. Running them in parallel gives feedback faster.